mdpow/templates/bar_charmm.mdp

```diff
@@ -1,7 +1,7 @@
 ; Gromacs mdp file
-; Template for MD Hydration FEP with OPLS-AA (part of POW)
+; Template for MD Hydration FEP with CHARMM (part of MDPOW)
 ;
-; grompp -pp -c ../../../MD_POSRES/md.gro -p ../../../top/system.top -f fep_opls.mdp -o md.tpr
+; grompp -pp -c ../MD_NPT/md.gro -p ../top/system.top -f fep.mdp -o md.tpr
```

Comment:
I don't really understand from which folder this command is executed, nor how `../../../top/system.top` can become `../top/system.top` without error, but if you have checked it it's fine.

Reply:
This command is never executed as written; it's more a reminder what one would be doing with the file. Technically it would require something like `../../../Equilibrium/water/MD_NPT` or something like it if run inside the MDPOW hierarchy. I just use `../MD_NPT` to indicate "get the starting structure from another directory that is not here". MD_POSRES does not even show up in MDPOW and was something from other manual simulation setups.

Reply:
I saw afterwards that the lines are commented out. Sorry to have bothered you with this.